src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Extensibility/ProjectExportProvider.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using System.Linq;
+using Microsoft.VisualStudio.Composition;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Extensibility
+{
+    /// <summary>
+    /// MEF component which has methods for consumers to get to project specific MEF exports
+    /// </summary>
+    [Export(typeof(IProjectExportProvider))]
+    [AppliesTo(ProjectCapability.CSharpOrVisualBasic)]
+    internal class ProjectExportProvider : IProjectExportProvider
+    {
+        private SVsServiceProvider ServiceProvider { get; set; }
```

Comment:
this is a property , not a field

---
In reply to: [74471471](https://github.com/dotnet/roslyn-project-system/pull/349#discussion_r74471471) [](ancestors = 74471471)